src/pages/languages.tsx

```diff
@@ -0,0 +1,20 @@
+import Head from "next/head";
+// import React from "react";
+
+
+
+///NOT SURE I NEED THIS COMPONENT!!!!!
```

Comment:
Probably delete this file for now? since we don't need it.

Reply:
agree